src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -640,56 +576,62 @@ def post(self, request, project):
             phone_id=phone_info.id,
             push_mode=phone_info.push_mode,
         )
-
         if init_phone_login is None:
             return Response("Phone 2fa failed", status=500)
 
         # success, return the new session context (add phone_id and push mode to the session context)
-        session_context["phone_id"] = phone_info.id
-        session_context["push_mode"] = phone_info.push_mode
-        encrypted_context = encrypt.encrypt_object(session_context, key)
-        return Response({"sessionContext": encrypted_context}, status=200)
+        data["phone_id"] = phone_info.id
+        data["push_mode"] = phone_info.push_mode
+        return Response(data, status=200)
+
+
+class TwoFactorAuthSessionContextSerializer(serializers.Serializer):
+    auth_key = serializers.CharField(min_length=1, required=True)
+    session_id = serializers.CharField(min_length=1, required=True)
+    scnt = serializers.CharField(min_length=1, required=True)
+    phone_id = serializers.CharField(min_length=1, required=False)
+    push_mode = serializers.CharField(min_length=1, required=False)
 
 
 class AppStoreConnect2FactorAuthSerializer(serializers.Serializer):
     """Input validation for :class:`AppStoreConnect2FactorAuthEndpoint."""
 
+    sessionContext = TwoFactorAuthSessionContextSerializer(required=True)
     code = serializers.CharField(max_length=10, required=True)
     useSms = serializers.BooleanField(required=True)
-    sessionContext = serializers.CharField(min_length=1, required=True)
 
 
 class AppStoreConnect2FactorAuthEndpoint(ProjectEndpoint):
     """Completes the 2FA iTunes login, returning a valid session.
 
     ``POST projects/{org_slug}/{proj_slug}/appstoreconnect/2fa/``
 
-    The request most contain the code provided by the user as well as the ``sessionContext``
+    The request must contain the code provided by the user as well as the ``sessionContext``
     provided by either the :class:`AppStoreConnectStartAuthEndpoint`
     (``projects/{org_slug}/{proj_slug}/appstoreconnect/start/``) call or the
     :class:`AppStoreConnectRequestSmsmEndpoint`
     (``projects/{org_slug}/{proj_slug}/appstoreconnect/requestSms/``) call:
```

Comment:
I am getting no `sessionContext` in the response when requesting SMS and this is causing de `2fa` request to fail